pulp_python/app/models.py

```diff
@@ -105,19 +110,23 @@ class PythonImporter(Importer):
     TYPE = 'python'
     projects = models.TextField(null=True)
 
-    def _fetch_inventory(self):
+    def _fetch_inventory(self, base_version):
         """
         Fetch existing contentunits in the repository.
 
+        Args:
+           base_version (pulpcore.plugin.models.RepositoryVersion): the latest pre-existing
+             version or None if one does not exist.
```

Comment:
I was thinking s/base_version/version/ since "base" doesn't really have relevance here. 

I think the summary line should be:
"Fetch the content units of the specified version"
and the `version` line should be: "Version to fetch content units from"